cookbooks/oceanic_extension/oceanic_extension.prm

```diff
@@ -0,0 +1,230 @@
+    set Surface velocity projection = normal 
+  end
+end
+
+# Velocity on boundaries
+# The boundary velocities are characterized by a free-slip on the base and outflow 
+# at a rate of 1 cm/year on the model sides. As no inflow is prescribed to balance the outflow, 
+# the average free surface elevation will decrease over time and the total mass of the system will decrease.
+subsection Boundary velocity model
+  set Tangential velocity boundary indicators = bottom
+  set Prescribed velocity boundary indicators = left x: function, right x:function 
+  subsection Function
+    set Variable names      = x,y
+    set Function constants  = cm=0.01, year=1
+    set Function expression = if (x<200e3 , -0.5*cm/year, 0.5*cm/year); 0; 
+  end
+end
+
+
+# Number and name of compositional fields
+# Three compositional fields are tracked during the simulation. 
+# The first represents strain accrued during brittle (plastic) deformation. 
+# The initial values of this field are used to localize deformation in the center of the model away from the boundaries. 
+# The remaining two compositional fields represent distinct lithologies for the oceanic crust and mantle lithosphere.
+subsection Compositional fields
+  set Number of fields = 3
+  set Names of fields = plastic_strain, oceanic_crust, mantle_lithosphere
+end
+
+
+# Spatial domain of different compositional fields
+# JBN - expand documenation here
+subsection Initial composition model
+  set Model name = function
+  subsection Function
+  # The function rand_seed(seed_number) will also produce the same random value at each point when the parameter file is run.
+  # "seed_number" is an integer and changing this value will produce a different (reproducible) random number. 
+  # A distinct random number can be produced each time the simulation is run by replacing rand_seed(seed_number) with rand().
+    set Variable names      = x,y
+    set Function expression = if(x>150.e3 && x<250.e3, 0.5 + rand_seed(0)*1.0, 0);  \
+			                        if(y>=93.e3, 1, 0); \
+                              if(y<93.e3 && y>-100.e3, 1, 0); 
+  end
+end
+
+
+# Temperature boundary conditions
+# Prescriibed temperature at bottom and top boundaries with values from the initial temperature field
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators = bottom, top
+  set Model name = initial temperature
+end
+
+# Initial temperature field
+# JBN - expand documentation here
+subsection Initial temperature model
+  set List of model names = function
+  subsection Function
+    set Variable names = x,y
+    set Function constants = ymax=100e3, ageop=1.26e15, Tm=1573, Ts=273, kappa=1.0101e-6
+    set Function expression = (Ts + (Tm-Ts)*(1-erfc((ymax-y)/(2*sqrt(kappa*ageop)))))
+  end
+end
+
+
+# Material model
+subsection Material model
+  set Model name = visco plastic
+
+  # JBN - add comment here about what this parameter does and why we choose "harmonic average" (or a different value)
```

Comment:
```suggestion
  # The `Material averaging` parameter determines how material properties are averaged over each element. Using `harmonic average` will produce the smoothest and most stable solution, although this will also act to reduce sharp variations in material properties at interfaces.
```